### PR TITLE
Using environment values for actinia templates

### PR DESCRIPTION
**Pull request by anikaweinmann**
Environment values should be able to be used inside actinia templates

- the environment depending values will be set via environment variables with the prefix `TEMPLATE_VALUE_`, then the plugin will use these values to set the values for the actinia module; e.g.
  - env variable: `TEMPLATE_VALUE_DB_CONNECTION`
  - and in the actinia module template `{{ db_connection }}`
  - the default env variable value can be overwritten by setting `db_connection` in the process chain of the actinia module
- by requesting an actinia module the plugin has to check if the parameter is set as environment variable. If yes:  
  - change `optional=True`
  - add to `description` "default value exist for this installation"

Additional, fixing for GRASS GIS modules with only one parameter.

### CODE REVIEW

File: actinia_module_plugin/core/modules/actinia_common.py [hunk 1]
```diff
@@ -389,6 +397,21 @@ def transformPlaceholder(self, placeholderCollector):
             self.vm_params.append(exe_param)
 
 
+def set_env_param_to_optional(params):
+    """
+    This function changes in a list with parameters the 'optional' value to
+    True and add a comment to the parameter 'description' is the parameter is
```
Comment (mmacata) [inline actinia_module_plugin/core/modules/actinia_common.py:403]:
```suggestion
    True and add a comment to the parameter 'description' if the parameter is
```

File: docker/actinia-module-plugin-test/Dockerfile [hunk 2]
```diff
@@ -30,9 +33,10 @@ COPY docker/actinia-module-plugin-test/actinia-module-plugin-test.cfg /etc/defau
...
 COPY . /src/actinia-module-plugin/
 
+
 WORKDIR /src/actinia-module-plugin/
 
 RUN chmod a+x tests_with_redis.sh
 RUN python3 setup.py install
 
-RUN ./tests_with_redis.sh
+# RUN ./tests_with_redis.sh
```
Comment (mmacata) [inline docker/actinia-module-plugin-test/Dockerfile:42]: Was this change done on purpose?

**Review by mmacata — APPROVED** [review]
Thanks! Cool! Just found a typo, else pls merge!

File: docker/actinia-module-plugin-test/Dockerfile (hunk 2 above)
Comment (anikaweinmann) [reply docker/actinia-module-plugin-test/Dockerfile]: No. Thanks

**Pull request merged**